README.md

```diff
@@ -1,13 +1,49 @@
-# Mongo API Project
+Mongo API Project 🍃
 
-Replace this readme with your own information about your project. 
+This project's main goal was to use MongoDB to build a database to store data, as well as, retrieve that data and use it to produce a RESTful API. Also it should be modelled and manipulated using mongoose.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
-## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Planning & What I learned 🧩
 
-## View it live
+The first step of this project was actually to read through MongoDB's documentation, install it and connect it to MongoDB Atlas, so my API could be deployed to Heroku.
+After everything was up and running I started looking at the data - I chose the songs dataset - and started planning some possible endpoints. Since we needed to use mongoose to model and manipulate the data, I then went ahead and create the mongoose model and started to build my endpoints with mongoose operators like find() & findOne()
+I had some problem with the resourcing on Herokup and was finally able to solve it with some help.
+Learned a lot about error handling!
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+
+
+Endpoints 💫
+
+/
+
+Root: Welcome page
+
+/topsongs
+
+Shows all songs in the dataset
+
+/topsongs/tracks?genre=:genre
+/topsongs/tracks?length=:length
+/topsongs/tracks?popularity=:popularity
+
+Shows the songs that match totally or partially the searched genre, song length and popularity respectively.
+
+/topsongs/tracks/:id
+
+Shows a single song based on that song's id parameter.
+
+
+
+Tech ⚡️
+
```

Comment:
Great read me file where you describe your working process and what you have learned. 👌

I couldn't get the endpoint genre to work. 
/topsongs/tracks?genre=:genre

I see also in the code that you never made a enpoint for genre. On the other hand there is an endpoint for bpm so maybe you just mixed them up?